src/entity-manager/EntityManager.ts

```diff
@@ -1187,10 +1189,12 @@ export class EntityManager {
     /**
      * Finds first entity by a given find options.
      * If entity was not found in the database - returns null.
+     * @param {Date} [timestamp] A timestamp to get datasets at a specific time.
      */
     async findOne<Entity extends ObjectLiteral>(
         entityClass: EntityTarget<Entity>,
         options: FindOneOptions<Entity>,
+        timestamp?: Date,
```

Comment:
please make this parameter part of the `FindOneOptions` object and name it better, so everyone understand what this "timestamp" is for.

Reply:
Good idea. This parameter is now part of the `FindOneOptions` with the name `validAt`.